lib/galaxy/managers/histories.py

```diff
@@ -215,7 +215,8 @@ def __init__( self, app ):
             'contents_url',
             # 'hdas',
             'empty',
-            'size', 'nice_size',
+            'size',
+            # 'nice_size',
```

Comment:
does this change the response to API calls like "GET /api/histories/{id}" in some cases?